.github/ISSUE_TEMPLATE/rule.md

```diff
@@ -0,0 +1,66 @@
+---
+name: 'Rule'
+about: Create new rule or update existing
+title: ''
+labels: 'engine'
+assignees: ''
+---
+
+
+<!-- Use this template if you want to create a new rule or add to an
+existing rule.
+
+If you are reporting a bug or problem, please use the bug template instead.
+
+Replace italicized text -->
+
+### Rule Description
+
+**Purpose and goal:** _one-line description of rule applicability (e.g., “This rule applies to any content that through CSS is displayed as a result of :hover”)_
+
+**Relevent requirement:** _Link to WCAG SC_
```

Comment:
I think the link should be to the IBM Requirement, such as 
https://www.ibm.com/able/requirements/requirements/#1_4_13

that way this is immediate info on the toolkit level

Reply:
incorporated